client/post-editor/controller.js

```diff
@@ -115,6 +115,7 @@ module.exports = {
 
 		function startEditing( siteId ) {
 			context.store.dispatch( setEditorPostId( postID ) );
```

Comment:
> Do you think it would be used in addition to or in place of the existing setEditorPostId ? 
I was thinking as a replacement on the controller, but I'm still wondering if I should keep the `setEditorPostId` for the specific usecase when saving a new post with ID yet, we'll have to update the ID without "starting a new editing session".

Let me try to put that in place, to have more context for discussion